scripts/languageserver/main.jl

```diff
@@ -1,34 +1,36 @@
-if VERSION < v"0.6.0-rc1" || VERSION >= v"0.7-"
-    error("VS Code julia language server only works with julia 0.6.")
-end
-
 try
+    if VERSION < v"1.0.0"
+        error("VS Code julia language server only works with julia 1.0.0+")
+    end
+
     if length(Base.ARGS) != 3
         error("Invalid number of arguments passed to julia language server.")
     end
 
-    conn = STDOUT
+    conn = stdout
     (outRead, outWrite) = redirect_stdout()
 
     if Base.ARGS[2] == "--debug=no"
         const global ls_debug_mode = false
     elseif Base.ARGS[2] == "--debug=yes"
         const global ls_debug_mode = true
     end
+    const global ls_debug_mode = true
 
-    push!(LOAD_PATH, joinpath(dirname(@__FILE__), "packages"))
-    push!(LOAD_PATH, Base.ARGS[1])
+    # push!(LOAD_PATH, joinpath(dirname(@__FILE__), "packages"))
+    # push!(LOAD_PATH, Base.ARGS[1])
```

Comment:
@KristofferC Is ``LOAD_PATH`` still a thing on 1.0? Essentially we just want to force the running instance to load things from our own private package directory.

Reply:
I would really like to avoid an install step... VS Code extensions are xcopy, really, and we would have to handle adding an install step ourselves, which will take time and add complexity, so ideally I would prefer to just stick with the submodules approach, I don't really see any downside to that (at least as long as we don't need any packages that have a ``build.jl``).

Reply:
Unfortunately StaticLint needs to be built before use, any ideas for getting around this?

Reply:
Hm, maybe that code could just go into an ``__init__`` function?

Reply:
@KristofferC could we include a Project but have it already instantiated?

Reply:
You would need to play with `DEPOT_PATH` when starting VSCode and loading your deps but it should be technically possible.